ext/tags.go

```diff
@@ -22,6 +22,9 @@ var (
 
 	// PeerPort records port number of the peer
 	PeerPort = uint16Tag("peer.port")
+
+	// SamplingPriority determines the priority of sampling this Span.
+	SamplingPriority = uint32Tag("sampling.priority")
```

Comment:
I see. Would make more sense (in `Go`) to not expose the constants directly but via an interface:

```go
type Uint16Tag interface{
  Set(opentracing.Span, value uint16)
}
var PeerPort Uint64Tag = uint64Tag("peer.port")
```